### ISSUE TITLE
tcc instructions don't make it clear that -run is to run C source as a C script.

### ISSUE THREAD

**Issue by experimentech** [vowstar/k210-linux-nommu#12]
The title pretty much says it. tcc -run executes C source as a script instead of compiling it.

**Comment by vowstar** [comment]
> The title pretty much says it. tcc -run executes C source as a script instead of compiling it.

No, tcc -run not a C language interpreter. It compile C script in memory and then execute it.
